packages/terra-search-field/src/SearchField.jsx

```diff
@@ -304,6 +309,7 @@ class SearchField extends React.Component {
           icon={Icon}
           isIconOnly
           isCompact
+          ref={this.searchBtnRef}
```

Comment:
Button supports[ `refcallback` prop](https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/button/about#button-props) to provide an callback ref to the dom. we can use `refcallback` to access dom.
```jsx
  <Button
    ------
    ------
    refCallback={(ref) => { this.searchBtnRef.current = ref; }}
  />
```

Reply:
Changed here: f4db4098cfb69d72c9f50e870151647eae859479